### DISCUSSION TITLE
FFmpeg在web端的潜力

### DISCUSSION THREAD

**Discussion by quink-black** [WebAV-Tech/WebAV#35] [General]
除了基础的demux/mux，decoder/encoder，还有什么其他的功能？

也包括可以有，但FFmpeg还没实现，期望能够实现的功能。

**Comment by hughfenghen** [comment]
@quink-black 
Web 开发者视角，ffmpeg.wasm 两个大问题：
1. 不支持 GPU 硬件加速编解码
2. 不支持流式数据处理

能解决任意一个都是值得庆祝的事情
